tools/sup-network/Makefile

```
@@ -0,0 +1,50 @@
+ifneq (${HAB_SUP_TEST_NETWORK_LOG_SERVICE},)
+	log_service := ${HAB_SUP_TEST_NETWORK_LOG_SERVICE}
+else
+	log_service := rando
+endif
+
+ifneq (${HAB_SUP_TEST_NETWORK_SIZE},)
+	peers := ${HAB_SUP_TEST_NETWORK_SIZE}
+else
+	peers := 3
+endif
+
+HAB_SUP_TEST_NETWORK_PEER_NAME ?= rando_1
+
+up: build CTL_SECRET
+	@docker build -t hab-sup-network .
+	@docker-compose up -d --scale rando=${peers}
+
+down:
+	@docker-compose down
+	@rm -f CTL_SECRET
+
+start:
+	@docker-compose start
+
+stop:
+	@docker-compose stop
+
+logs:
+	@docker-compose logs -f ${log_service}
+
+console:
+	@docker exec -it $$(docker ps -aqf name=${HAB_SUP_TEST_NETWORK_PEER_NAME}) /bin/bash
```

Why not use the [`shell`](https://www.gnu.org/software/make/manual/html_node/Shell-Function.html#Shell-Function) function?
```suggestion
	@docker exec -it $(shell docker ps -aqf name=${HAB_SUP_TEST_NETWORK_PEER_NAME}) /bin/bash
```

---

Mostly due to my own ignorance about `Makefile`s and the best way to do things.  Is there an advantage to using `shell` over `$$`?

---

I'm not totally sure. I don't think I'd ever seen the `$$` before, but `$(shell …)` is fairly common and easier to look up.